### DISCUSSION TITLE
Trojan detected in latest update

### DISCUSSION THREAD

**Discussion by LordDorn** [SnosMe/awakened-poe-trade#796] [Q&A] [answered]
In the last update in December 2022, anti-virus protection began to work: **PDM:Trojan.Win32.Generic** detected, was this done intentionally or was there some kind of error?

![image](https://user-images.githubusercontent.com/42858770/206867589-dc22406e-ee6f-4917-86c5-3d405fb973be.png)

**Comment by SnosMe** [comment, marked as answer]
That's a classic false positive, when AV takes hash and tries to know how popular it was to be launched.
If you have time report as false positive.
All builds are being made on GitHub servers, I can't upload files with malware.
